projects/aca-playwright-shared/src/api/user-actions.ts

```diff
@@ -106,15 +57,7 @@ export class UserActions {
    * @param permanent Delete permanently, without moving to the trashcan? (default: true)
    */
   async deleteSites(siteIds: string[], permanent: boolean = true) {
-    try {
-      if (siteIds && siteIds.length > 0) {
-        for (const siteId of siteIds) {
-          await this.sitesApi.deleteSite(siteId, { permanent });
-        }
-      }
-    } catch (error) {
-      Logger.error(`\n [User Actions] - deleteSites failed error : ${error}`);
-    }
+      return this.sitesApi.deleteSites(siteIds,permanent);
```

Comment:
you don't need this api here as it belongs to sitesApi... just make your tests invoke sitesApi instead

Reply:
done